.github/workflows/dockerfile-validation.yml

```
@@ -0,0 +1,40 @@
+name: Dockerfile Validation
+
+on:
+  push:
+    branches: [ main, master ]
+    paths:
+      - 'Dockerfile'
+      - '.github/workflows/dockerfile-validation.yml'
+  pull_request:
+    branches: [ main, master ]
```

```suggestion
    branches: [ main ]
```

We don't have master branch